### DISCUSSION TITLE
Integrate with Remix

### DISCUSSION THREAD

**Discussion by heriparid** [refinedev/refine#2145] [Q&A] [answered]
Hi,
I want to build full stack web that can be integrated with [Remix](https://remix.run/) framework
Is there any tutorial that might help me for getting started?
Thanks.

**Comment by omeraplak** [comment, marked as answer]
Hey @heriparid 👋,
Short answer, yes it works with Remix. We've worked on Remix support before, but we had issues with Navigation. If you want, you can review this PR, I think it may be useful for you ⚡️

https://github.com/pankod/refine/pull/1712

**Reply by heriparid** [reply]
Well, can't wait to have "refine-remix-router" to be available.
